Cargo.toml

```diff
@@ -1,7 +1,8 @@
 
 [workspace]
 members = [
-    "icelake"
+    "icelake",
+    "rest_api"
```

Comment:
I'm thinking about using tools written in rust like https://github.com/juhaku/utoipa. But it's ok to get started first.